tck/pom.xml

```diff
@@ -39,6 +39,30 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.eclipse.microprofile.reactive-streams-operators</groupId>
+            <artifactId>microprofile-reactive-streams-operators-api</artifactId>
+            <version>${microprofile-reactive-streams-operators.version}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <!-- TODO: DELETE START -->
+        <!--
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-rs-mp-client</artifactId>
+            <version>3.3.6</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.eclipse</groupId>
+            <artifactId>yasson</artifactId>
+            <version>1.0.7</version>
+        </dependency>
+    -->
+        <!-- TODO: DELETE END -->
```

Comment:
Do you want to keep this here ?

Reply:
Good catch @phillip-kruger - at first, I was "cheating" by testing the CXF implementation in the same project as the TCK.  I've removed it.